starwars/src/App.js

```diff
@@ -1,19 +1,41 @@
-import React from 'react';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import axios from "axios";
+import CharacterList from "./components/CharacterList";
+import styled from "styled-components";
 
 const App = () => {
-  // Try to think through what state you'll need for this app before starting. Then build out
-  // the state properties here.
+  const starWarsApi = "https://swapi.co/api/people";
 
-  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
-  // side effect in a component, you want to think about which state and/or props it should
-  // sync up with, if any.
+  const [characterArray, setCharacterArray] = useState([]);
+
+  useEffect(
+    () =>
+      axios
+        .get(starWarsApi)
+        .then(response => {
+          setCharacterArray(response.data.results);
+        })
+        .catch(error => {
+          console.log("Error");
```

Comment:
I know nothing was mentioned about this in the instructions of the project, but maybe, mayyybe next time implement a feature which would let users know if there was an error while trying to fetch data from the API? A simple `alert(error.message);` or maybe a separate, styled, component just for the error rendering on the screen?